ui/app/routes/settings/tokens.js

```diff
@@ -0,0 +1,12 @@
+// @ts-check
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class SettingsTokensRoute extends Route {
+  @service store;
+  model() {
+    return {
+      authMethods: this.store.findAll('auth-method'),
+    };
```

Comment:
`nit`: If this is the only property returned on the model couldn't we just write:

```
async model() {
  return this.store.findAll('auth-method')
}
```

Reply:
For sure, but I kind of like the explicitness of asking for `this.model.authMethods` on a tokens page.

(If you asked me in a vacuum what `this.model` would iterate on a page called `tokens`, I would not assume it would be `auth-method`s, so IMO better to make it explicit)